source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRPresentationViewController.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ACRPresentationViewController.m
+//  ADCIOS
+//
+//  Created by jwoo on 5/31/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ACRPresentationViewController.h"
+#import "ACRViewController.h"
+
+@implementation ACRPresentationViewController
+
+- (id) init:(NSString*) str{
+    self = [super init];
+    
+    if(self){
+        _jsonStr = nil;
+        _jsonStr = str;
+    }
+    
+    return self;
+}
+
+- (void) viewDidLoad {
+
+    [super viewDidLoad];
+    
+    if(self.jsonStr){
+        ACRViewController* acVc = [[ACRViewController alloc] init: self.jsonStr];
```

Comment:
from my understanding without this wrapper, user needs to have at least one file that has .mm extension  or needs a compilation flag that link c++ std lib, but now you raised this point, i think you might be correct because now i'm using framework, adding the compilation flag may not necessary. is the compilation flag not needed?

---
In reply to: [122553246](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122553246) [](ancestors = 122553246,122552582,122549111,122548933)